src/main/scala/org/broadinstitute/hail/expr/AST.scala

```diff
@@ -213,6 +213,37 @@ object AST extends Positional {
         null
     }
   }
+
+  def evalComposeNumericArray[T](ec: EvalContext, subexpr: AST)(g: (IndexedSeq[T]) => Any)
```

Comment:
actually, I think this doesn't handle missing properly.  Give me a little time to think on it